erpnext/public/js/financial_statements.js

```diff
@@ -267,11 +267,10 @@ function get_filters() {
 	let fy_filters = filters.filter((x) => {
 		return ["from_fiscal_year", "to_fiscal_year"].includes(x.fieldname);
 	});
-	let fiscal_year = erpnext.utils.get_fiscal_year(frappe.datetime.get_today(), false, true);
+	let fiscal_year = erpnext.utils.get_fiscal_year(frappe.datetime.get_today(), false, false);
```

Comment:
Doesn't this `false` cause an unnecessary error message?

Reply:
Yes, indeed, the python implementation is suboptimal, as well.

Reply:
The implementation now looks much better.